webapi/src/test/scala/org/knora/webapi/e2e/v2/ValuesRouteV2E2ESpec.scala

```diff
@@ -170,6 +170,45 @@ class ValuesRouteV2E2ESpec extends E2ESpec {
     }
 
     "The values v2 endpoint" should {
+        "get the latest version of a value, given its UUID" in {
```

Comment:
Are these the same tests as for `ResourcesResponderV2Spec`, just for the values route this time?

Reply:
I don't understand the question. The resources route doesn't accept value UUIDs, and this PR doesn't add anything to `ResourcesResponderV2Spec`.

Reply:
I think it's just that the resources responder is used internally to get specific values so the tests were added to `ResourcesResponderV2Spec`. But the values route v2 is called in the API so the tests go into `ValuesRouteV2E2ESpec`.

Reply:
Sorry, I got confused. Yes, that's right.

Reply:
no problem, I first had to understand that v1 and v2 handling of values is different :-)